manager/state/raft/membership/cluster.go

```diff
@@ -180,7 +181,7 @@ func (c *Cluster) ReplaceMemberConnection(id uint64, oldConn *Member, newConn *M
 		return ErrIDNotFound
 	}
 
-	if oldConn.Conn != oldMember.Conn {
+	if !force && oldConn.Conn != oldMember.Conn {
```

Comment:
The old version of this function is kind of like a check-and-set. It won't replace the connection if it has already been replaced. This is to prevent multiple `sendToMember` goroutines from fighting each other. One could bounce the connection, then this kills the old connection, which causes another goroutine to have an RPC failure, which causes it to replace the connection again, and so on.

It's different when we see an address change in raft. Then we can't do best-effort. We need to replace the connection no matter what. So that's why there's a force option.